code/modules/urist/items/clothes/belt.dm

```diff
@@ -19,7 +19,11 @@
  	/obj/item/device/flashlight,
  	/obj/item/stack/cable_coil,
  	/obj/item/device/analyzer,
- 	/obj/item/device/mmi
+ 	/obj/item/device/mmi,
+	/obj/item/device/integrated_electronics/debugger,
```

Comment:
Adding /obj/item/device/integrated_electronics instead of every subtype will work just fine.

Reply:
Got it, thanks.